src/status_im/contexts/onboarding/create_profile/view.cljs

```diff
@@ -144,7 +144,7 @@
                                      (rf/dispatch
                                       [:show-bottom-sheet
                                        {:content (fn []
-                                                   [method-menu/view on-change-profile-pic])
+                                                   [picture-picker/view on-change-profile-pic false])
```

Comment:
I would advise passing the props in a map. It will reduce the time needed to alter the arguments when adding or removing when the view grows.